roletools/events.py

```diff
@@ -265,7 +265,7 @@ async def give_roles(
         else:
             to_add = set(member.roles)
 
-        log.debug("Atomic role assignment %s", atomic)
+        log.verbose("Atomic role assignment %s", atomic)
```

Comment:
```suggestion
        log.debug("Atomic role assignment %s", atomic)
```
This one would actually be useful for debugging a users settings.